test/analysis_ee_ZH_cfg.py

```diff
@@ -274,20 +333,15 @@ def next():
         print usage
         sys.exit(1)
             
-        
     loop = Looper( 'looper', config,
                    nEvents=10,
                    nPrint=1,
                    timeReport=True)
     
-    simulation = None
     for ana in loop.analyzers: 
-        if hasattr(ana, 'display'):
-            simulation = ana
-    display = getattr(simulation, 'display', None)
-    simulator = getattr(simulation, 'simulator', None)
-    if simulator: 
-        detector = simulator.detector
+        if hasattr(ana, 'display') and ana.display == True:
+            display = True # will display first event in a loop
```

Comment:
after merging with the new version of my master to be able to use this code with the new edm, I'm getting the following and no display:

```
[test]$ ipython analysis_ee_ZH_cfg.py 0 
Couldn't import dot_parser, loading of dot files will not be possible.
Jet clusterizer in ee mode, ee_kt, E sorting
Info in <TPad::Print>: file looper_2/heppy.analyzers.PapasEventPlotter.PapasEventPlotter_1/event_0_sim_rec.png has been created
#--------------------------------------------------------------------------
#                         FastJet release 3.1.0
#                 M. Cacciari, G.P. Salam and G. Soyez                  
#     A software package for jet finding and analysis at colliders      
#                           http://fastjet.fr                           
#                                                                             
# Please cite EPJC72(2012)1896 [arXiv:1111.6097] if you use this package
# for scientific work and optionally PLB641(2006)57 [hep-ph/0512210].   
#                                                                       
# FastJet is provided without warranty under the terms of the GNU GPLv2.
# It uses T. Chan's closest pair algorithm, S. Fortune's Voronoi code
# and 3rd party plugin jet algorithms. See COPYING file for details.
#--------------------------------------------------------------------------

```